filter.js

```diff
@@ -132,39 +131,39 @@ const sentEventClick = (eventName) => {
         renderElement(eventList);
       }
     };
-    const generatePrice = (e) => {
-      let price = e.cost;
-      if (Number.isFinite(price)) {
-        price = e.cost.toFixed(2).replace(/\./g, ",");
-        price += " zł";
-        return price;
+
+    const displayPriceWithCurrency = (singleEvent) => {
```

Comment:
btw. protip - takie funkcje możesz spokojnie wyciągnąć poza ten ten listener. tam choćby na top czy obojętne gdzie ale w jedno miejsce z innymi. Normalnie z webpackiem mógłbyś to nawet wyciąghnąć na sam szczyt, ale wtedy bylyby w globalnym zasięgu a tego się unika :)